R/sitevisitpeak-data.R

```diff
@@ -196,7 +196,7 @@ containsOutsideUncertainty <- function(data) {
 #'@export
 svpQualifiersTable <- function(data, table){
   #Construct List of all qualifiers
-  qualifiersList <- data.frame(unlist(data$readings$associatedIvQualifiers, recursive=FALSE))
+  qualifiersList <- do.call("rbind", data$readings$associatedIvQualifiers)
```

Comment:
one of the tests is failing because `data$readings$associatedIvQualifiers` is NULL. You'll need to safeguard against that. I suggest:

```r
if(!isEmptyOrBlank(data$readings$associatedIvQualifiers)){
    qualifiersList <- do.call("rbind", data$readings$associatedIvQualifiers)
} else {
    qualifiersList <- data.frame()
}
```